pkgs/development/compilers/ghc/8.8.2.nix

```diff
@@ -86,12 +86,12 @@ let
 
 in
 stdenv.mkDerivation (rec {
-  version = "8.8.1.20191211";
+  version = "8.8.2";
   name = "${targetPrefix}ghc-${version}";
 
   src = fetchurl {
-    url = "https://downloads.haskell.org/ghc/8.8.2-rc1/ghc-${version}-src.tar.xz";
-    sha256 = "1gl4fzakjbhd94v1saxmr9sfzgk22m1b95jq51rxm93b2g4cixl4";
+    url = "https://downloads.haskell.org/ghc/8.8.2/ghc-${version}-src.tar.xz";
+    sha256 = "1gl4fzakjbhd94v1saxmr9sfzgk22m1b95jq51rxm93b2g4cixl5";
```

Comment:
How can the build pass if the checksum is wrong???

Reply:
Fixed-output derivation.  If you already have a result with correct hash, no building happens and the result is re-used.  (So you don't need to download tarballs when you already have them, for example.)

Reply:
Still the bot could be friendly and tell me this fact (a warning maybe, or a GitHub comment)?

Reply:
Got it: `nix-prefetch-url --unpack` :-)